powermeter/holdtimer.py

```
@@ -0,0 +1,54 @@
+        self.held_values: Optional[List[float]] = None
+        self.hold_until: float = 0.0
+
+    def wait_for_message(self, timeout=5):
+        """Pass through to wrapped powermeter."""
+        return self.wrapped_powermeter.wait_for_message(timeout)
+
+    def get_powermeter_watts(self) -> List[float]:
+        current_time = time.time()
+
+        if self.held_values is not None and current_time < self.hold_until:
+            # Still within the hold window – return the held value
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)